examples/auth/solidstart/SUMMARY.md

```
@@ -0,0 +1,194 @@
+    │       ├── client.ts
+    │       ├── server.ts
+    │       └── middleware.ts
+    └── routes/
+        ├── index.tsx
+        ├── login.tsx
+        ├── logout.tsx
+        └── protected.tsx
+```
+
+### Modified Files
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)